pkg/adapter/serializer/targetComponent.go

```diff
@@ -0,0 +1,67 @@
+package serializer
+
+import (
+	"errors"
+	"fmt"
+	"github.com/cloudogu/cesapp-lib/core"
+	"github.com/cloudogu/k8s-blueprint-operator/pkg/domain"
+	"github.com/cloudogu/k8s-blueprint-operator/pkg/util"
+)
+
+type TargetComponent struct {
+	// Name defines the name of the component including its namespace, f. i. "official/nginx". Must not be empty.
+	Name string `json:"name"`
+	// Version defines the version of the dogu that is to be installed. Must not be empty if the targetState is "present";
+	// otherwise it is optional and is not going to be interpreted.
```

Comment:
```suggestion
	// Version defines the version of the component that is to be installed. Must not be empty if the targetState is "present";
	// otherwise it is optional and is not going to be interpreted.
```